source/slang/slang-check-constraint.cpp

```diff
@@ -284,7 +284,7 @@ namespace Slang
         // These seem more reasonable to have influence constraint solving, since it could
         // conceivably let us specialize a `X<T> : IContainer` to `X<Int>` if we find
         // that `X<T>.IndexType == T`.
-        for( auto constraintDeclRef : getMembersOfType<GenericTypeConstraintDecl>(genericDeclRef) )
+        for( auto constraintDeclRef : getMembersOfType<GenericTypeConstraintDecl>(genericDeclRef, MemberFilterStyle::All) )
```

Comment:
One case of using `MemberFilterStyle::All` slipped through the cleanup change. Probably not worth editing to fix unless you really want to.